src/three-render-objects.js

```diff
@@ -298,7 +306,23 @@ export default Kapsule({
     }
 
     state.renderer.setSize(state.width, state.height);
-    state.camera.aspect = state.width/state.height;
+
+    if (state._cameraType === 'perspective') {
+      state.camera.aspect = state.width/state.height;
+    } else {
+      const aspect = state.width / state.height;
+      const height_ortho = depth * 2 * Math.atan(
+        THREE_JS_PERSPECTIVE_CAMERA_FOV_Y_DEFAULT * (Math.PI / 180) / 2
+      );
+      const width_ortho  = height_ortho * aspect;
+
+      state.camera.left = width_ortho / -2;
+      state.camera.right = width_ortho / 2;
+      state.camera.top = height_ortho / 2;
+      state.camera.bottom = height_ortho / -2;
+      state.camera.near = THREE_JS_PERSPECTIVE_CAMERA_NEAR_DEFAULT;
```

Comment:
This appears to be the current default `near` value of orthographic camera, so maybe there's no need to define it. It can always be overridden upstream.

Reply:
done